src/main/java/uk/gov/justice/probation/courtcaseservice/service/CourtCaseService.java

```diff
@@ -73,21 +97,15 @@ public CourtCaseEntity createOrUpdateCase(String caseId, CourtCaseEntity courtCa
         existingCase.setBreach(courtCaseEntity.getBreach());
         existingCase.setDefendantName(courtCaseEntity.getDefendantName());
         existingCase.setDefendantAddress(courtCaseEntity.getDefendantAddress());
-
-        log.info("Court case updated for case {}", courtCaseEntity.getCaseNo());
+        existingCase.setPnc(courtCaseEntity.getPnc());
```

Comment:
I aded these fields to be set because they weren't orginally. No tests existed to send updates based on those fields.